src/pipeline/parse.js

```diff
@@ -172,7 +188,7 @@ exports.getHTML = (paths, ast, components, datasets, template) => {
     componentClasses[key] = require(components[key])
   })
 
-  require(resolve.sync('react-syntax-highlighter', { basedir: paths.DEFAULT_COMPONENTS_DIR }));
+  requireHighlightJS(ast, paths);
```

Comment:
Can we just do `exports.getHighlightJS(ast, paths, true);` directly here and get rid of `requireHighlightJS`?